src/lava/magma/core/process/ports/ports.py

```diff
@@ -339,7 +339,7 @@ class RefPort(AbstractRVPort, AbstractSrcPort):
     RefPort to a Var via the connect_var(..) method."""
```

Comment:
"Shared-memory-based communication can have side-effects and should therefore be used with caution."

Would it be possible to elaborate on this? As a user, I will be alerted after reading this sentence, but will not know what to do with it.